vms/avm/vm.go

```diff
@@ -482,6 +469,16 @@ func (vm *VM) Linearize(_ context.Context, stopVertexID ids.ID, toEngine chan<-
 	// chainVM has been initialized. Traffic will immediately start being
 	// handled asynchronously.
 	vm.Atomic.Set(vm.network)
+
+	go func() {
+		err := vm.state.Prune(&vm.ctx.Lock, vm.ctx.Log)
```

Comment:
Should we put a sentinel somewhere so we don't redo this prune every time?

Reply:
If there is nothing to prune it will immediately terminate.